### PR TITLE
New networks

### PR DESCRIPTION
**Pull request by Deciphersoul**
Adding the remaining Onlymoons supported networks not on testnet.

### CODE REVIEW

File: src/networks/index.ts [hunk 1]
```diff
@@ -6,3 +6,16 @@ export { default as bsctest } from './97-bsctest'
...
+export { default as canto } from './7700-canto'
+export { default as conflux } from './1030-conflux'
+export { default as dogechain } from './2000-dogechain'
+export { default as ethw } from './10001-ethw'
+export { default as fvm } from './314-fvm'
+export { default as kekchain } from './420420-kekchain'
+export { default as poochain } from './20202021-poochain'
+export { default as pulse } from './369-pulse'
+export { default as metis } from './1088-metis'
+export { default as milkomeda } from './2001-milkomeda'
```
Comment (floticerus) [inline src/networks/index.ts:21]: can you put these in order of the chain id? 😇

**Review by floticerus — CHANGES_REQUESTED** [review]
One change plz 😇

File: src/networks/index.ts (hunk 1 above)
Comment (Deciphersoul) [reply src/networks/index.ts]: Done!

**Review by floticerus — APPROVED** [review]
🎉

**Pull request merged**